docs/fastdds/dds_layer/topic/contentFilteredTopic/defaultFilter.rst

```diff
@@ -0,0 +1,358 @@
+.. include:: ../../../../03-exports/aliases.include
+.. include:: ../../../../03-exports/aliases-api.include
+.. include:: ../../../../03-exports/roles.include
+
+
+.. _dds_layer_topic_contentFilteredTopic_default_filter:
+
+The default SQL-like filter
+===========================
+
+Filter expressions used by :ref:`dds_layer_topic_contentFilteredTopic` API may use a subset of SQL syntax, extended with
+the possibility to use program variables in the SQL expression. |br|
+This section shows this default SQL-like syntax and how to use it.
+
+
+* :ref:`default_sql_filter_grammar`
+* :ref:`default_sql_filter_like`
+* :ref:`default_sql_filter_match`
+* :ref:`default_sql_filter_type_comparisons`
+* :ref:`default_sql_filter_example`
+
+.. _default_sql_filter_grammar:
+
+Grammar
+-------
+
+The allowed SQL expressions are defined with the BNF-grammar below.
+
+The following conventions are made:
+
+- "Terminals" are quoted.
+- ``TOKENS`` are typeset in code block and small caps.
+
+.. productionlist::
+    Expression: FilterExpression
+    FilterExpression: Condition
+    Condition: Predicate | Condition "AND" Condition | Condition "OR" Condition | "NOT" Condition | "(" Condition ")"
+    Predicate: ComparisonPredicate | BetweenPredicate
+    ComparisonPredicate: FIELDNAME RelOp Parameter | Parameter RelOp FIELDNAME | FIELDNAME RelOp FIELDNAME
+    BetweenPredicate: FIELDNAME "BETWEEN" Range | FIELDNAME "NOT BETWEEN" Range
+    RelOp: "=" | ">" | ">=" | "<" | "<=" | "<>" | "!=" | `like` | `match`
+    Range: Parameter "AND" Parameter
+    Parameter: INTEGERVALUE | CHARVALUE | FLOATVALUE | STRINGVALUE | ENUMERATEDVALUE | PARAMETER
+
```

Comment:
We should include a note stating that both the capitalized and lowercase special words are supported: `AND`, `and`, `OR`, `or`, `LIKE`, etc.

Reply:
done on 2eaf78ee